openedx/features/idea/helpers.py

```diff
@@ -12,3 +16,43 @@ def upload_to_path(instance, filename, folder):
         filename=filename,
         folder=folder
     )
+
+
+def pretty_date(time):
+    """
+    :param time
+    :return: string object in "time ago" format
+    """
+    now = datetime.utcnow().replace(tzinfo=utc)
+    diff = now - time
+    second_diff = diff.seconds
+    day_diff = diff.days
+
+    if day_diff == 0:
```

Comment:
https://docs.djangoproject.com/en/3.0/ref/contrib/humanize/

Reply:
https://github.com/philanthropy-u/edx-platform/blob/cc2ca35413b18f56e55fd5a14a21ce07c6dd75c2/openedx/features/teams/serializers.py#L96

Reply:
changed it bro